hakcbot_commands.py

```diff
@@ -90,6 +90,14 @@ def hw(self):
     def laptop(self):
         return 'i use this laptop --> dell e7250 12.5in, i5-5300U 2.3ghz, 8g ram, 256 ssd'
 
+    @cs.cmd('iobound', THREE_MIN)
+    def laptop(self):
```

Comment:
can you make the function definition the same as the string name in the decorator? :)

Reply:
Yep.  Silly copy/paste error.